client/app/components/dashboards/dashboard-widget/RestrictedWidget.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Widget from './Widget';
+
+function RestrictedWidget(props) {
+  return (
+    <Widget {...props} className="d-flex justify-content-center align-items-center widget-restricted">
+      <div className="t-body scrollbox">
+        <div className="text-center">
+          <h1><span className="zmdi zmdi-lock" /></h1>
+          <p className="text-muted">
+            {'This widget requires access to a data source you don\'t have access to.'}
```

Comment:
Any reason this ain't plain html?
```suggestion
            This widget requires access to a data source you don't have access to.
```

Reply:
I was having an eslint warning on this one:
![eslint-1](https://user-images.githubusercontent.com/3356951/64698769-df0fd980-d479-11e9-9ab7-bf6e9fd1fb41.png)

It's either that or
```jsx
<p>This widget requires access to a data source you don&quot;t have access to.</p>

Reply:
Use `&apos;` 👍